src/main/kotlin/org/malachite/estella/commons/models/people/User.kt

```diff
@@ -20,6 +20,10 @@ data class User(
         this.password = password ?: ""
     }
 
+    /*
+        TODO [ES-187] - User should be data class, final form of password should be passed to default constructor
```

Comment:
Ogólnie to że jest to w setterze zapewnia nam że nie ma opcji byśmy mieli nie zaenkryptowane hasło zapisane w bazie